src/jaybenne/transport.cpp

```diff
@@ -225,6 +233,8 @@ TaskStatus TransportPhotons(MeshData<Real> *md, const Real t_start, const Real d
             }
           }
           rng_pool.free_state(rng_gen);
+          // reset fraction of particles that make it census
+          ppack_r(b, ph::fraction(), n) = 1.0;
```

Comment:
If the fraction gets reset to 1 at census, does it need to be stored as a particle property?